sdk/spring/spring-cloud-azure-native-configuration/src/main/java/com/azure/core/AzureCoreHints.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core;
```

Comment:
Shouldn't the root package match the artifact id, like `com.azure.spring.native.configuration`?